docs/index.mdx

```diff
@@ -5,10 +5,17 @@ title: '❓ What is Etherlink?'
 import Figure from '@site/src/components/Figure';
 
 Etherlink is an EVM-compatible layer-2 blockchain powered by [Tezos Smart Rollup](https://tezos.com/developers/smart-rollups/) technology.
+It is a permissionless network that users can run transactions on and deploy smart contracts to.
+It has independent governance, built-in MEV protection, and many other features for developers.
```

Comment:
We don't yet have built in MEV protection built in with the current beta so I think it's best to drop or make it explicit that's in coming in a future version.

Reply:
Removed mention of MEV protection.